subgraph/tests/keys.test.ts

```diff
@@ -0,0 +1,161 @@
+import {
```

Comment:
Hum, wouldn't it be more robust to use an actual deployment on hardhat?

Reply:
yes, these are unit tests with mocks just to make sure we do things properly at the unit level then we need integration tests with hardhat and actual contracts deployed.